app/inputs/patternfly_checkbox_input.rb

```diff
@@ -4,12 +4,14 @@ class PatternflyCheckboxInput < Formtastic::Inputs::BooleanInput
   delegate :tag, to: :template
 
   def to_html
+    checkbox = tag.div(class: 'pf-c-check') do
+      hidden_field_html + input + label + description
+    end
+
+    return checkbox if options[:nude]
```

Comment:
Hmm... is that needed? I don't notice any changes if I drop this.

Reply:
I makes the resulting HTML exactly like Patternfly 4, but like you say, it doesn't affect either the layout or the tests so I will remove it.